### PR TITLE
reflecting k8s-sig migration

### PR DESCRIPTION
**Pull request by jimangel**
Almost resolves: https://github.com/kubernetes/website/issues/13904

/cc @remyleone for fixes in French localization (https://github.com/kubernetes/website/search?q=kubernetes-incubator%2Freference-docs&type=Code)

Related: https://github.com/kubernetes/org/issues/1158

### CODE REVIEW

**Comment by kbhawkey** [comment]
@jimangel , looks good.
If time, would you remove the last link (federation) in the `What's next` section on this page?
https://deploy-preview-16632--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/contribute-upstream/
/lgtm

**Comment by jimangel** [comment]
> If time, would you remove the last link (federation) in the `What's next` section on this page?
> https://deploy-preview-16632--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/contribute-upstream/

Done. Thanks!

File: content/en/docs/contribute/generate-ref-docs/contribute-upstream.md [hunk 1]
```diff
@@ -240,7 +240,6 @@ You are now ready to follow the [Generating Reference Documentation for the Kube
 * [Generating Reference Documentation for the Kubernetes API](/docs/contribute/generate-ref-docs/kubernetes-api/)
 * [Generating Reference Docs for Kubernetes Components and Tools](/docs/home/contribute/generated-reference/kubernetes-components/)
 * [Generating Reference Documentation for kubectl Commands](/docs/home/contribute/generated-reference/kubectl/)
-* [Generating Reference Documentation for the Kubernetes Federation API](/docs/home/contribute/generated-reference/federation-api/)
```
Comment (kbhawkey) [inline content/en/docs/contribute/generate-ref-docs/contribute-upstream.md]: Thanks! Blast it, another small nit, line 233, extra `to`:
`You are now ready to follow the [Generating Reference Documentation for the Kubernetes API](/docs/contribute/generate-ref-docs/kubernetes-api/) guide to to generate the`
/lgtm

**Comment by jimangel** [comment]
Should be all set now @kbhawkey - thanks for the review!

**Pull request merged**